src/main/resources/jupyter/init-actions.sh

```diff
@@ -17,6 +17,26 @@ set -e -x
 # Retry 4/5 exited 1, retrying in 16 seconds...
 # Retry 5/5 exited 1, no more retries left.
 #
+# Array for new relic instrumentation
```

Comment:
can you remove the mention of newrelic since the metrics are not sent to newrelic